src/session/session_api.c

```diff
@@ -167,6 +222,41 @@ __session_clear(WT_SESSION_IMPL *session)
 	session->nhazard = 0;
 }
 
+/*
+ * __session_close_cursors --
+ *	Close all cursors in a list.
+ */
+static int
+__session_close_cursors(WT_SESSION_IMPL *session, WT_CURSOR_LIST *cursors)
+{
+	WT_CURSOR *cursor, *cursor_tmp;
+	WT_DECL_RET;
+
+	/* Close all open cursors. */
+	WT_TAILQ_SAFE_REMOVE_BEGIN(cursor, cursors, q, cursor_tmp) {
+		if (F_ISSET(cursor, WT_CURSTD_CACHED))
+			/*
+			 * Put the cached cursor is an open state
+			 * that allows it to be closed.
+			 */
+			WT_TRET_NOTFOUND_OK(cursor->reopen(cursor, false));
+		else
+			/*
+			 * Notify the user that we are closing the cursor handle
+			 * via the registered close callback.
+			 */
+			if (session->event_handler->handle_close != NULL &&
```

Comment:
The `if` should be joined with the `else` above and then you can move the comment inside the conditional.

Reply:
Thanks, done.